examples/src/main/java/com/yugabyte/examples/UniformLoadBalanceExample.java

```diff
@@ -11,15 +11,15 @@
 import java.util.*;
 
 public class UniformLoadBalanceExample {
-  private static String controlUrl = "";
-  private static HikariDataSource hikariDataSource;
+  public static String controlUrl = "";
+  public static HikariDataSource hikariDataSource;
   static Scanner scanner = new Scanner(System.in);
   static List<Connection> borrowConnections = new ArrayList<>();
 
   public static void main(String[] args) {
     try {
-      Boolean VERBOSE = (args[0].equals("1") ? true : false);
-      Boolean INTERACTIVE = (args[1].equals("1") ? true : false);
+      Boolean verbose = (args[0].equals("1") ? true : false);
```

Comment:
What if args is empty. It will throw NPE. Also this is enough: 

args[0].equals("1") -- It will either return true or false.

so check for length of args and then the above check.

Reply:
the logic for this is in the shell scripts, the script will either pass "0" or "1" for both the arguments, but will not leave the arguments.